config.yml

```diff
@@ -15,4 +15,5 @@ cachet:
   component_id: 1
   #metric_id: 1
   create_incident: true
+  update_status: true
```

Comment:
Actually, `create_incident` and `update_status` are mutually exclusive. The way you made it, if `create_incident` is left as true the `update_status` is ignored. This should be instead an enum.